docs/source/endpoints/rules.md

```diff
@@ -0,0 +1,167 @@
+  "property=og:title": "Content rules"
+  "keywords": "Plone, plone.restapi, REST, API, ContentRules, Rules"
+---
+
+(rules)=
+
+# Content rules
+
+```{note}
+These docs are generated by code tests, therefore you will see some `test` rules appear here.
+```
+
+Available content rules in a Plone site can be listed and queried by accessing the `/@content-rules` endpoint on any context.
+Access requires the ManageContentRules permission.
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/rules_get.req
+```
+
+```{literalinclude} ../../src/plone/restapi/tests/http-examples/rules_get.resp
+:language: http
+```
+
+The API consumer can assign, unassign , enable, disable,
+(disable) apply to subfolders any of the rules available in the portal.
+
+| Verb    | URL              | Action                                             |
+| ------- | ---------------- | -------------------------------------------------- |
+| `POST`  | `/@content-rules/{rule-id}`| Add rule to context                                          |
+| `GET`   | `/@content-rules/`         | Get aquired, assignable and assigned rules                      |
+| `PATCH` | `/@content-rules/` with RAW Body | enable/disable/(disable) apply to subfolders/move down/up |
+| `DELETE`| `/@content-rules/` with RAW Body | Unassign rule on context                                  |
+
+
+## Assigning a new Content rule with `POST`
+
+To assign a content rule to a context, send a `POST` request to the `<context>/@content-rules` endpoint:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/rules_add.req
+```
+
+Response:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/rules_add.resp
+:language: http
+```
+
+## Getting content rule on a context with `GET`
+
+To get content rules assigned/assignable/aquired on a context, send a `GET`
+request to the `<context>/@content-rules` endpoint:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/rules_get.req
+```
+
+Response:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/rules_get.resp
+:language: http
+```
+
+## Changing content rules on a context with `PATCH`
+
+To make changes on content rule assignments on a context, send a `PATCH`
+request to the `<context>/@content-rules` endpoint:
+
+### Apply on subfolders
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../../src/plone/restapi/tests/http-examples/rules_apply_subfolders.req
+```
+
+Response:
+
+```{literalinclude} ../../../src/plone/restapi/tests/http-examples/rules_apply_subfolders.resp
+:language: http
+```
+
+### Disable apply on subfolders
```

Comment:
The request and response are identical to those from the previous subsection. Maybe I don't know the difference between "Apply" and "Disable Apply", but I would expect the examples to reflect whatever that difference may be.

Reply:
This was a very good catch! Should be fixed now.

Reply:
Yup, looks good.